api/views.py

```diff
@@ -54,3 +54,8 @@ class AssetLogViewSet(ModelViewSet):
     permission_classes = [IsSecurityUser]
     authentication_classes = (FirebaseTokenAuthentication,)
     http_method_names = ['get', 'post']
+
+    def perform_create(self, serializer):
+        sec_user = SecurityUser.objects.get(id=self.request.user.id)
+        if self.request.user:
```

Comment:
you could do `sec_user = self.request.user.securityuser`, then remove the conditional since a user has to be present to access this serializer.